apstra/api_design_templates_test.go

```diff
@@ -956,7 +956,7 @@ func TestAllOverlayControlProtocols(t *testing.T) {
 
 func TestRackBasedTemplateMethods(t *testing.T) {
 	ctx := context.Background()
-	var apiVersion string
+	var apiVersionString string
```

Comment:
I think that there's no need to rename these variables to apiVersionString. There's no other kind of apiVersion in this particular test

Reply:
It made sense to me at the time, but maybe only in an intermediate stage as I was making changes.

In any rate, it'll be mooted by the constraints-based PR that this one lays groundwork for. Expect this string to disappear altogether soon.